tests/test_capitelli_integrals.cpp

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
Why did you add this file?  Everything here is in `test_collision_integrals.cpp` I think.

Reply:
I don't know. Probably a leftover of the rebase. I'll delete it.